include/ddc/ddc.hpp

```diff
@@ -35,3 +37,41 @@
 #if defined(DDC_BUILD_PDI_WRAPPER)
 #include "ddc/pdi.hpp"
 #endif
+
+#if ginkgo_AVAIL
+#include <ginkgo/ginkgo.hpp>
+#endif
```

Comment:
The following function already assumes `Ginkgo` is available, no need to `ifdef` only that part
```suggestion
#include <ginkgo/ginkgo.hpp>
```

Reply:
I will try to support the case with gingko_AVAIL=false, to use DDC without splines

Reply:
Then I would advise a name related to splines. I would also suggest to move source code out of this header. I think it is better that it remains only a list of private headers.

Reply:
commit 78b7c484c3f10d8997ee02101d0e28b1a7ac8bf6 introduces cmake options to disable FFT or splines tests/examples/benchmarks. It also loaded CUDA toolkit, Ginkgo etc... only if required.

Reply:
Not sure where to put the code which is in ddc.hpp then... Ginkgo may not be specific to splines even if its the case atm.

Reply:
Also, this code would be removed if https://github.com/ginkgo-project/ginkgo/pull/1358 is merged (but no update for a long time)